robolectric/src/test/java/org/robolectric/shadows/ShadowICUTest.java

```diff
@@ -1,22 +1,47 @@
 package org.robolectric.shadows;
 
+import android.app.Activity;
 import android.os.Build;
+import android.os.Bundle;
+import android.widget.DatePicker;
+import android.widget.LinearLayout;
 
 import libcore.icu.ICU;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.robolectric.Robolectric;
 import org.robolectric.TestRunners;
 import org.robolectric.annotation.Config;
+import org.robolectric.util.ActivityController;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 @RunWith(TestRunners.MultiApiWithDefaults.class)
 @Config(sdk = {
-    Build.VERSION_CODES.LOLLIPOP })
+  Build.VERSION_CODES.LOLLIPOP,
+  Build.VERSION_CODES.LOLLIPOP_MR1})
```

Comment:
AH, no my bad... we don't support M yet, I'm just adding it in the branch :-) This is great as is. Thanks.